Documentation/helm-ceph-cluster.md

```diff
@@ -32,20 +32,68 @@ Rook currently publishes builds of this chart to the `release` and `master` chan
 
 The release channel is the most recent release of Rook that is considered stable for the community.
 
+**Before installing, review the values.yaml to confirm if the default settings need to be updated.
+The [defaults](https://github.com/rook/rook/tree/{{ branchName }}/cluster/charts/rook-ceph-cluster/values.yaml)
+are only an example and not likely to apply to your cluster.**
+
+The example install assumes you have created a values-override.yaml.
+
 ```console
 helm repo add rook-release https://charts.rook.io/release
-helm install --create-namespace --namespace rook-ceph rook-ceph --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster
+helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
+   --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster -f values-override.yaml
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the rook-operator chart and their default values.
+
+| Parameter             | Description                                                          | Default     |
+| --------------------- | -------------------------------------------------------------------- | ----------- |
+| `operatorNamespace`   | Namespace of the Rook Operator                                       | `rook-ceph` |
+| `configOverride`      | Cluster ceph.conf override                                           | <empty>     |
+| `toolbox.enabled`     | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md) | `false`     |
+| `toolbox.tolerations` | Toolbox tolerations                                                  | `[]`        |
+| `toolbox.affinity`    | Toolbox affinity                                                     | `{}`        |
+| `monitoring.enabled`  | Enable Prometheus integration, will also create necessary RBAC rules | `false`     |
+| `cephClusterSpec.*`   | Cluster configuration, see below                                     | See below   |
+
+
+### Ceph Cluster Spec
+
+The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
+For the full list, see the [Cluster CRD](ceph-cluster-crd.md) topic.
+
+### Existing Clusters
+
+If you have an existing CephCluster CR that was created without the helm chart and you want the helm
+chart to start managing the cluster:
+
+1. Extract the `spec` section of your existing CephCluster CR and copy to the `cephClusterSpec`
+   section in `values-override.yaml`.
+
+2. Add the following annotations and label to your existing CephCluster CR:
+
 ```
+  annotations:
+    meta.helm.sh/release-name: rook-ceph-cluster
+    meta.helm.sh/release-namespace: rook-ceph
+  labels:
+    app.kubernetes.io/managed-by: Helm
```

Comment:
:+1: I haven't tested this for Rook specifically, but I have done this in production and confirm it works for other charts